test/test_THDM_py_interface.py.in

```diff
@@ -0,0 +1,378 @@
+#!/usr/bin/env python
```

Comment:
Can you please rename the file to `test_THDM_python_interface.py.in`? I find this a bit more expressive instead of just having `_py_interface.py.in`. :)

Reply:
Is done, no worries.

Reply:
Thanks!